### PR TITLE
scheduler package

### PR DESCRIPTION
**Pull request by shamsimam**

### CODE REVIEW

File: waiter/src/waiter/scheduler/scheduler.clj [hunk 1]
```diff
@@ -8,7 +8,7 @@
 ;;       The copyright notice above does not evidence any
 ;;       actual or intended publication of such source code.
 ;;
-(ns waiter.scheduler
+(ns waiter.scheduler.scheduler
```
Comment (sradack) [inline waiter/src/waiter/scheduler/scheduler.clj]: Why not leave everything you moved to `waiter.scheduler.scheduler` in `waiter.scheduler`?  Prior art:

https://github.com/clojure/clojure/tree/master/src/clj/clojure, `core.clj` and `core` folder

Comment (pschorf) [reply waiter/src/waiter/scheduler/scheduler.clj]: I agree with @sradack

Comment (shamsimam) [reply waiter/src/waiter/scheduler/scheduler.clj]: Undid move of `scheduler.clj`

File: waiter/src/waiter/scheduler/shell_scheduler.clj [hunk 2]
```diff
@@ -8,7 +8,7 @@
 ;;       The copyright notice above does not evidence any
 ;;       actual or intended publication of such source code.
 ;;
-(ns waiter.shell-scheduler
+(ns waiter.scheduler.shell-scheduler
```
Comment (sradack) [inline waiter/src/waiter/scheduler/shell_scheduler.clj]: Now that we have `scheduler` in the namespace, the `-scheduler` in `shell-scheduler` seems redundant.  What do you think?

Comment (shamsimam) [reply waiter/src/waiter/scheduler/shell_scheduler.clj]: I'm ok renaming it to `shell.clj`. Or do you prefer a different name?

Comment (shamsimam) [reply waiter/src/waiter/scheduler/shell_scheduler.clj]: Renamed to `shell.clj`

**Comment by shamsimam** [comment]
RTM

**Comment by shamsimam** [comment]
Can we get this merged?

**Pull request merged**